### PR TITLE
Linux install docs

### PR DESCRIPTION
**Pull request by palmtree5**
## Before PR
Please make sure that you've completed the following steps

 - [x] Added frontmatter header to your doc based on [this guide](/README.MD)
 - [x] Formatted the doc to fit jekyll requirements based on [this guide](/README.MD)
 - [x] Added link to a sidebar based on [this guide](/README.MD)

## Doc info
Please provide a required info on your PR

 * Category: guides
 * Subject: Installation guides for various Linux distros (see also #78 )
 * Author: @Kowlin and myself
 
There are currently three categories available: troubleshooting, guides and custom cog documentation, please specify one of those in the list above
Subject might be anything, but we recommend sticking to 3-4 words

### CODE REVIEW

**Comment by Kowlin** [comment]
leaving this for @Twentysix26 due the fact that its a big change from what we currently have here. And many thanks for doing the work on this palm ;)

**Comment by Chovin** [comment]
few comments:

- [x] submenu/page of the collection of linux distros so that the sidebar is just Windows/Mac/Linux
- [x] fix main page dead link

File: red/red_install_debian.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+{% include callout.html content="**Warning**: For safety reasons, DO NOT install Red with a root user. Instead, make a new one." type="danger" %}
+
+## Installing the pre-requirements
+
+```
+apt install nano
+nano /etc/apt/sources.list
+```
+
+add `deb http://httpredir.debian.org/debian jessie-backports main contrib non-free` on its own line at the end of the file. Save and exit (Ctrl+X, y, Enter)
```
Comment (Chovin) [inline red/red_install_debian.md]: simplify this section?

**Comment by palmtree5** [comment]
Fixed the dead main page link with a new page called `red_install_linux` that just lists links to all the guides. I tried creating a category for all the linux guides but clicking it doesn't do anything other than redirect back to the main page

**Comment by palmtree5** [comment]
Ok, category for all linux guides isn't happening. It'll just be a page linked in the sidebar and on the main page with all the guides listed there

**Comment by palmtree5** [comment]
Added docs for OpenSUSE

**Pull request merged**